.changeset/chatty-ravens-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+"hardhat": patch
+---
+
+Added logic to make sure that solc default import callback is not being used
```

Comment:
I don't think we need to include a changeset for this; there's no user-facing change, it's more of an internal quality improvement.